vscode/src/services/LocalStorageProvider.ts

```diff
@@ -37,6 +37,7 @@ class LocalStorage implements LocalStorageForModelPreferences {
     private readonly MODEL_PREFERENCES_KEY = 'cody-model-preferences'
     private readonly AUTO_EDITS_BETA_ENROLLED = 'cody-auto-edit-beta-onboard'
     private readonly DEVICE_PIXEL_RATIO = 'device-pixel-ratio'
+    public readonly STORAGE_SIZE_BIG = 50_000 * 1024 // 50,000 KB
```

Comment:
```suggestion
    public readonly CHAT_STORAGE_SIZE_MAX = 50_000 * 1024 // 50,000 KB
```

nit: STORAGE_SIZE_BIG  makes this sounds like the upper limit for everything in the storage when we are only using it for chat history. Maybe name it to make it clear what this storage size is used for specifically, something like CHAT_STORAGE_SIZE_MAX? Not a blocker though.

Reply:
@abeatrix how about CHAT_STORAGE_SIZE_BIG? I think big is better than max since the user can still use chat when the storage exceeds this size, it's just not recommended.